src/components/AccountCircleButton.tsx

```diff
@@ -1,45 +1,31 @@
 import React from 'react'
-import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import { StyleProp, ViewStyle } from 'react-native'
 import { HomeEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
-import Touchable from 'src/components/Touchable'
 import AccountCircle from 'src/icons/AccountCircle'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { Spacing } from 'src/styles/styles'
+import { TopBarIconButtonV2 } from 'src/navigator/TopBarIconButtonV2'
 
 interface Props {
   style?: StyleProp<ViewStyle>
   size?: number
   testID?: string
 }
 
-export default function AccountCircleButton({ style, size, testID }: Props) {
+export default function AccountCircleButton({ testID, size, style }: Props) {
   const onPress = () => {
     ValoraAnalytics.track(HomeEvents.account_circle_tapped)
     navigate(Screens.ProfileMenu)
   }
 
   return (
-    <View style={styles.container}>
-      <Touchable
-        testID={testID}
-        onPress={onPress}
-        style={[style, styles.button]}
-        borderRadius={Spacing.Thick24}
-      >
-        <AccountCircle size={size} />
-      </Touchable>
-    </View>
+    <TopBarIconButtonV2
+      icon={<AccountCircle />}
+      testID={testID}
+      onPress={onPress}
+      size={size}
```

Comment:
I think the size prop should actually be passed in with the icon component, like it was done previously, for example `<AccountCircle size={size} />`.

Looking at `TopBarIconButtonV2` it looks like it does take a size prop but is not used.

Same goes for the other files where `TopBarIconButtonV2` was added.